html/index.js

```diff
@@ -128,7 +128,8 @@ function filterTestCasesBasedOnStateHandler (tableId, tableName, state, mandator
   }
   const tableIdClean = tableId.replace(/#/g, '')
   const table = document.getElementById(tableIdClean)
-  const elements = table.getElementsByClassName('accordion-item')
+  const elements = table.getElementsByTagName('rh-accordion-header')
```

Comment:
Fix formatting issues: I fixed the current formatting issues in this branch
https://github.com/aabughosh/parser/compare/improvment...edcdavid:parser:fix-formatting?expand=1